packages/cursorless-org/src/content/enablement-group.mdx

```diff
@@ -0,0 +1,80 @@
+export const meta = {
+  title: "Cursorless Enablement Group",
+  description:
+    "Help enable the adoption of Cursorless by speeding up its development.",
+};
+
+# {meta.title}
+
+## 🚀 {meta.description}
+
+Cursorless, an open-source spoken programming language, lets users eliminate the need for a cursor entirely. In other words, software engineers can code at the speed of sound (increased productivity) and all computer users can reduce strain on their wrists to prevent injury (preventative healthcare).
```

Comment:
I'll wordsmith this one.

Reply:
Sounds good! Are you OK with launching as it is or plan to wordsmith prior to launcing?